frontend/packages/dev-console/console-extensions.json

```diff
@@ -186,6 +186,20 @@
       ]
     }
   },
+  {
+    "type": "dev-console.import/environment",
+    "properties": {
+      "imageStreamName": "nodejs",
+      "imageStreamTags": ["14-ubi8", "14-ubi8-minimal", "12-ubi8", "latest"],
+      "environments": [
+        {
+          "key": "NPM_RUN",
+          "label": "%devconsole~Run command%",
```

Comment:
Same as above, a description would be helpful, or?

```suggestion
          "label": "%devconsole~Run command%",
          "description": "%devconsole~Optional parameter for Node.js applications to define the npm run <command>. It is saved as environment variable NPM_RUN of the BuildConfig.%",
```

Reply:
Updated.
![1](https://user-images.githubusercontent.com/20013884/139849678-b87f565f-bfb2-4ab0-8f46-6efc6603196b.png)